e2e/__tests__/__snapshots__/failures.test.ts.snap

```diff
@@ -786,8 +786,11 @@ FAIL __tests__/snapshot.test.js
 
     Snapshot name: \`failing snapshot 1\`
 
-    Snapshot: "bar"
-    Received: "foo"
+    - Snapshot  1 -
+    + Received  1 +
+
+    - bar
+    + foo
```

Comment:
For this case, where it's a string that was 1 line and will again be 1 line, this information is pretty useless. Can we not display it?

Reply:
Yes, I will deal with comparison of two one-line strings as a special case for the concise labeled format, because it seems likely for `toThrowErrorMatchingSnapshot` assertions.

Reply:
Maybe with a clearer labeling (other comment thread) it's actually fine and worth keeping for consistency?

Reply:
Tim, good point about prioritizing **consistency** higher than a more concise special case.

If most reports have **differences** when snapshot assertions fail, and if next pull request changes the colors, some people might learn to scan quickly which console output is from snapshots versus other assertions, and more intuitively choose which decision path.

For example, the `e2e` tests for this pull request had 3 snapshots to update and 1 `toMatch` assertion which I needed to rewrite as 4 assertions.

To test this theory: think fast, what do you need to do for each of the following:

```text
Expected: bar
Received: foo
```

```text
Snapshot: bar
Received: foo
```

Reply:
I think I know what you mean, if there's lots of (correct) test failures I usually try to change non-snapshot tests first and then update snapshots once they're the only things still failing. Not sure how that's related to this comment thread but I'd definitely appreciate making it easier to tell them apart.

Reply:
See pictures in https://github.com/facebook/jest/pull/8982#issuecomment-536069809 of concise label format if neither string is multiline